dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/ServletFilter.java

```diff
@@ -125,7 +125,14 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
 					//SRTServletRequest allows us to get the WebAppDispatcher
 					SRTServletRequest srtServletRequest = (SRTServletRequest) servletRequest;
 					
-					httpRoute = resolveHttpRoute(srtServletRequest, contextPath);
+					//do spring check
+					if (MonitorAppStateListener.isSpringFeatureEnabled() && srtServletRequest.getAttribute("org.springframework.web.servlet.HandlerMapping.bestMatchingPattern") != null) {
```

Comment:
Should use `org.springframework.web.servlet.HandlerMapping.BEST_MATCHING_PATTERN_ATTRIBUTE`